### PR TITLE
CSS color: tweaks

### PR DESCRIPTION
**Pull request by estelle**
* Update `color-gamut` media descriptor with links and glossary terms
* updated `HSL()` color function, changing the `hsla()` legacy example to be a modern versus legacy example and making the complementary colors paragraph note clearer (and fixed a typo). Also, updated the see also.
* updated the `<hex-color>` page to add alpha to the example and make see also conform to how they are written on other pages.

Part of https://github.com/openwebdocs/project/issues/194 (will be making numerous small-ish pull requests to enable smaller chunk PR reviews)

### CODE REVIEW

**Comment by bsmth** [comment]
Thanks, Estelle. I'll take a look at this soon 👍🏻

File: files/en-us/web/css/@media/color-gamut/index.md [hunk 1]
```diff
@@ -57,6 +57,9 @@ p {
 
 ## See also
 
-- [color()](/en-US/docs/Web/CSS/color_value/color) function to specify colors in a defined colorspace.
-- [@media](/en-US/docs/Web/CSS/@media) at-rule that is used to specify the color-gamut expression.
+- [`color()`](/en-US/docs/Web/CSS/color_value/color) function to specify colors in a defined colorspace.
```
Comment (bsmth) [inline files/en-us/web/css/@media/color-gamut/index.md:60]: Do we add a Glossary entry (color space) in the See also according to the guidelines, or is it enough to have in the article?

File: files/en-us/web/css/color_value/hsl/index.md [hunk 2]
```diff
@@ -294,38 +294,62 @@ div.comma-separated {
 
 {{EmbedLiveSample("legacy_syntax_comma-separated_values", "100%", 150)}}
 
-### Legacy syntax: hsla()
+### Legacy versus modern syntax
 
-The `hsla()` syntax is an alias for `hsl()`.
+The example demonstrates how `hsla()` syntax is an alias for `hsl()`; both are supported using both modern and legacy syntaxes.
```
Comment (bsmth) [inline files/en-us/web/css/color_value/hsl/index.md:299]: Do you want to reiterate "comma-separated" as legacy again for good measure?

```suggestion
The example demonstrates how the `hsla()` syntax is an alias for `hsl()`; both are supported using both modern and legacy (comma-separated) syntaxes.
```

File: files/en-us/web/css/color_value/hsl/index.md [hunk 3]
```diff
@@ -294,38 +294,62 @@ div.comma-separated {
...
+body {
+  display: flex;
+  gap: 20px;
+}
+```
 
-div.hsl {
-  background-color: hsl(0 100% 50% / 50%);
+```css
+div.modern {
```
Comment (bsmth) [inline files/en-us/web/css/color_value/hsl/index.md:329]: not important for now: when skimming the example, I was looking for comments that hint on legacy syntax -> I see it's in the class name, but my reflex is to look for comments, maybe from syntax boxes that look like this:

```css
/* RGB (Red, Green, Blue) */
rgb(255 0 153)
```

File: files/en-us/web/css/hex-color/index.md [hunk 4]
```diff
@@ -35,60 +35,71 @@ A `<hex-color>` value can be used everywhere where a [`<color>`](/en-US/docs/Web
 
 ## Examples
 
-### Hexadecimal syntax for a fully opaque hot pink
+### Hexadecimal hot pink
+
+This example includes four hot pink squares, with fully opaque or semi-transparent backgrounds created using four different-length case-insensitive hex-color syntaxes.
```
Comment (bsmth) [inline files/en-us/web/css/hex-color/index.md:40]: nice 👍🏻

**Review by bsmth — APPROVED** [review]
One minor suggestion to look at, otherwise looking great! Thank you! 🎉

File: files/en-us/web/css/@media/color-gamut/index.md (hunk 1 above)
Comment (estelle) [reply files/en-us/web/css/@media/color-gamut/index.md:60]: there are two links to color space in the prose, which is not that long, so not needed here.

**Pull request merged**